ops/services/alerts/app_service_metrics/exceptions.tf

```diff
@@ -0,0 +1,53 @@
+# To do log-based alerts, we need to make sure we're actually collecting AppServiceConsoleLogs from the API
+data "azurerm_resource_group" "management" {
+  name = "prime-simple-report-management"
+}
+
+data "azurerm_log_analytics_workspace" "global" {
+  name                = "simple-report-log-workspace-global"
+  resource_group_name = data.azurerm_resource_group.management.name
+}
+
+resource "azurerm_monitor_diagnostic_setting" "collect_appserviceconsolelogs" {
+  name                       = "API App Service console logs"
+  target_resource_id         = var.app_service_id
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.global.id
+
+  log {
+    category = "AppServiceConsoleLogs"
+    enabled  = true
+
+    retention_policy {
+      days    = 7
+      enabled = true
+    }
+  }
+}
+
+# Add an alert for GraphQL query validation failures (more than 3 in a 5-minute window)
+resource "azurerm_monitor_scheduled_query_rules_alert" "graphql_query_validation_failures" {
+  name                = "graphql-query-validation-failures"
+  location            = "East US"
```

Comment:
Ugh. Why does Azure want the location half the time but not the other half? (And am I right in thinking that it's actually always required to be the location of the resource group anyway, so this is just them being jerks?)

Reply:
Your parenthetical made me go check out the TF resource and sure enough, the resource group has location as an output. Added a new commit to use that instead.